node-require/add.js

```diff
@@ -0,0 +1,5 @@
+function add(x,y) {
+  return parseInt(x) + parseInt(y);
+}
+
+module.exports.add = add;
```

Comment:
How would you adapt this to work with fractional numbers as well?

Reply:
![Screen Shot 2021-08-04 at 3 52 07 PM](https://user-images.githubusercontent.com/24905313/128260139-b5e92d75-5a6e-4ce0-909a-93516fefaeb8.png)